app/next-env.d.ts

```diff
@@ -1,4 +1,5 @@
 /// <reference types="next" />
+/// <reference types="next/types/global" />
```

Comment:
you need to update your local yarn packages. This was removed in the latest nextjs updage